src/components/attendee/ScheduleOverlay/helpers.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { MutableRefObject, RefObject } from "react";
+import classNames from "classnames";
+import { IntersectingButton } from "components/attendee/IntersectingButton";
+import { addDays } from "date-fns";
+import { range } from "lodash";
+
+import { ALWAYS_EMPTY_ARRAY } from "settings";
+
+import { ScheduledEvent } from "types/venues";
+
+import { isEventWithinDateAndNotFinished } from "utils/event";
+import { formatDayLabel } from "utils/time";
+
+import { Button } from "../Button";
+
+import CN from "./ScheduleOverlay.module.scss";
+
+const minWeekDaysScrollValue = 8;
+
+type WeekDayProp = {
+  dayDifference: number;
+  firstScheduleDate: Date;
+  selectedDayIndex: number;
+  liveAndFutureEvents: ScheduledEvent[];
+  isScheduleTimeshifted: boolean;
+  setSelectedDayIndex: React.Dispatch<React.SetStateAction<number>>;
+  allRefs?: MutableRefObject<RefObject<HTMLButtonElement>[]>;
+  updateIntersected?: (el: RefObject<HTMLButtonElement>) => void;
+};
+export const generatedWeekDays = ({
```

Comment:
This is basically a `<WeekdaysSomethingSomething>` component

Reply:
done [ede5a78](https://github.com/sparkletown/sparkle/pull/2921/commits/ede5a78fee222b03794b50d81d1b5bb452d05814)